src/events/pull/index.ts

```diff
@@ -75,8 +70,11 @@ async function checkPullFormat(context: Context, pullService: PullService) {
     sha: head.sha,
     state: reply.status === Status.Success ? "success" : "failure",
     target_url: "https://github.com/ti-community-infra/ti-community-bot",
-    description: reply.message,
-    context: "Sig Info File Format",
+    description:
+      reply.message.length > GITHUB_STATUS_DESC_MAX_LENGTH
+        ? reply.message.substr(0, 130) + "..."
```

Comment:
Maybe 130 is calculated by the `GITHUB_STATUS_DESC_MAX_LENGTH` constant will be better.

Reply:
Fixed.